src/components/Layout.re

```diff
@@ -147,7 +147,7 @@ module Header = {
                        clearAndPush("#explorer");
                      }}
                      className=Styles.whiteText>
-                     <S> "ANIMAL KINGDOM" </S>
+                     <S> "VIEW ALL WILDCARDS" </S>
```

Comment:
Agree, I think very few people tried to click the "animal kingdom" button.

Reply:
Marketplace? 
Otherwise I am happy

Reply:
Not sure about the connotations associated with marketplace? Maybe a future change.